core/services/relay/evm/mercury/v3/data_source.go

```diff
@@ -26,6 +27,13 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
 )
 
+var bidAskInvariantViolations = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Could we send this in EA telemetry instead of just recording a node metric?
https://github.com/smartcontractkit/chainlink/blob/develop/core/services/ocrcommon/telemetry.go